### ISSUE TITLE
Duplicate Row on auto_increment = 0 ID

### ISSUE THREAD

**Issue by jstrellner** [TablePlus/TablePlus#1806]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
MySQL 5.6.47

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
300

3. The steps to reproduce this issue:
Go to a table with multiple columns and an auto_increment configured that has data. Right click on a row, and click on _duplicate_ (it will create a new row with the auto_increment column empty and defaulted to "DEFAULT") then hit command+s to save it, and it will replace the auto_increment field as a 0 instead of the next number.

If instead of duplicating you just add a new row, the DEFAULT column DOES get changed to the next number.

**Comment by jstrellner** [comment]
If I do the process outlined above, but change it from DEFAULT two writing in NULL, it then gives me the next increment...

**Comment by jstrellner** [comment]
@huyphams I do not think this is an enhancement, it's a bug. Creating a new entry works as expected, duplicating an existing row does not.  Expected behavior would be for the auto_increment field to get assigned the next ID.

Also, confirmed this issue still exists in version 304.

**Comment by lorkel** [comment]
I'm still experiencing this too... any ideas on what is causing it (and how to resolve)?

**Comment by nathlrowe** [comment]
This is still an issue.
